src/main/java/com/softserveinc/ita/rozetka/utils/ConfigProperties.java

```
@@ -0,0 +1,27 @@
+package com.softserveinc.ita.rozetka.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigProperties {
+    private final Properties properties;
+
+    public ConfigProperties() {
+        properties = new Properties();
+        try {
+            FileInputStream fileInputStream = new FileInputStream("src/main/resources/config.properties");
+            properties.load(fileInputStream);
+        } catch (IOException e) {
+            e.printStackTrace();
```

do not hide exceptions